boards/shields/inventek_eswifi/inventek_eswifi.overlay

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2020 ATL Electronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * This file is intentionally empty. Please confer with shield documentation
```

Comment:
I am not a native English speaker, but to my ears the word `refer to` would work better here instead of `confer with`

Reply:
Fixed!